### DISCUSSION TITLE
User unable to move osu!lazer file directory

### DISCUSSION THREAD

**Discussion by msmcculley** [ppy/osu#28896] [Q&A]
Hello, [this post](https://osu.ppy.sh/community/forums/topics/1948974?n=1) is from the help forum.

The user apparently has two separate osu!lazer folders, one containing the client and a separate one containing what seems to be the realm files and caches. [image of what that looks like here](https://prnt.sc/MSnHtjqAPsB5). [the client folder looks like this](https://prnt.sc/WQYobpXS2j15)

This user is experiencing issues with moving their osu!lazer folder to a new directory. They have tried moving their installation through the in-game file moving button but it produces an unsatisfactory result. If they try and manually move the osu!lazer folder they get no good result and osu!lazer just makes a [new realm/cache folder in the same directory as before](https://prnt.sc/MSnHtjqAPsB5).

Not sure on the direction going forward with this type of behavior.

Thanks for your time.

**Comment by SupDos** [comment]
Are the lightshot screenshots not loading for anyone else?

Would probably be best to have the user use the export logs button in-game after trying to move the folder to try and figure out why it's not moving correctly

edit: also to note, if they moved the data folder manually, they can go back into lazer and use the file moving button to pick that folder - lazer should detect the folder contains data and swap to it without needing to move anything, might be worth having them try that?

**Comment by maor727** [comment]
Hi I am the user.
first of all some working screenshots:
Here is the [folder with the osu client ](https://media.discordapp.net/attachments/1253005080991502419/1263255038864916502/image.png?ex=6699916f&is=66983fef&hm=995fd935569e0610806a0554d3dc1f5cd390d8211a7e45d6a96d555dc7999d8e&=&format=webp&quality=lossless&width=835&height=287)
[The folder with all the maps etc](https://media.discordapp.net/attachments/1253005080991502419/1263255201746518057/image.png?ex=66999196&is=66984016&hm=dd10fda9ab1956793a28d220dc6af03983887a3ab9ad3bff6ad325b8f0df3189&=&format=webp&quality=lossless&width=821&height=521)

Now I try to export logs after trying to move the folder:
[compressed-logs.zip](https://github.com/user-attachments/files/16275213/compressed-logs.zip)
Also I think I should mention that I run osu as administrator when I do it, because my D folder asks for administration perms.

Hope you can help me out, I really want to enjoy lazer :)

[my post on osu website](https://osu.ppy.sh/community/forums/topics/1948974?n=1)

**Reply by SupDos** [reply]
> I run osu as administrator

This is most likely your problem, you shouldn't be running osu with admin perms as it will break things

As for why - assuming you only run it as admin when moving the folder over: once you restart the game without admin, it isn't able to read/write to the files in the D drive as they require admin perms, so it defaults to making new files in AppData.

I would recommend fixing the permissions issue on your D drive so that it doesn't require admin perms to write to it, then trying to move the folder over.

**Reply by maor727** [reply]
I am going to sleep now I will try tomorrow but it seems like its gonna work.
Thank you both for your time <3

**Reply by SupDos** [reply]
Also just to add, I noticed in your screenshot you have the client folder on your D drive as well
I'd suggest not moving this folder that has the osu! client, that should always be in AppData\Local\osulazer. It may make the updater confused and isn't officially supported. It will only ever take up ~600MB, your main storage concern would be the data folder which is what you can move via in game settings

**Reply by maor727** [reply]
Do I just manually move the osu client folder from D to C?

**Reply by SupDos** [reply]
I would just delete that folder and run the installer from the website, just to make sure nothing weird happens due to moving stuff around (especially since your permissions seem to be weird on the D drive)

Make sure the larger data folder with all your maps is stored safely somewhere, once you fix the permissions issue you should be able to select it in game and it will swap over to it
